client/components/tinymce/plugins/contact-form/dialog/dialog.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormButton from 'components/forms/form-button';
+
+const FormItem = React.createClass( {
+	onClick() {
+		this.props.onClick( this.props.index );
+	},
+
+	render() {
+		return <li>{ this.props.label } <a onClick={ this.onClick }>Remove</a></li>
```

Comment:
This is intentional to be a constant reminder that this component is entirely provisional, built for the only purpose of testing the state changes. There is another PR blocked by this one that replaces this component with one that do have translations.